solr/solrj/src/test/org/apache/solr/client/solrj/io/stream/MathExpressionTest.java

```diff
@@ -1719,10 +1719,10 @@ public void testMatrix() throws Exception {
     assertEquals(features.get(0).get(0), "col3");
     assertEquals(features.get(1).get(0), "col1");
 
-    assertTrue(tuples.get(0).getLong("f") == 2);
-    assertTrue(tuples.get(0).getLong("g") == 3);
-    assertTrue(tuples.get(0).getLong("h") == 1);
-    assertTrue(tuples.get(0).getLong("i") == 2);
+    assertEquals(2, (long) tuples.get(0).getLong("f"));
```

Comment:
I think so - I'll take a look

Reply:
Hmmm so `tuple.getLong` returns a `Long` because it also returns `null` sometimes. Changing to `long` can't be done for `tuple.getLong` since then `null` can't be returned. `3L` is still a `long` and can't be compared with `Long` apparently. I could change it to

```
assertEquals(Long.valueOf(3), tuple.getLong("myId"))
```

but that seems worse.

I don't have any other ideas right now :/